go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -300,9 +309,9 @@ func (c *addressManager) nodePrimaryAddrChanged() (bool, error) {
 		return false, err
 	}
 	// check to see if ips on the node differ from what we stored
-	// in addressManager
+	// in addressManager and it's an address that makes sense
```

Comment:
What  are the conditions for "makes sense".

Reply:
Fixed by making the comment more specific